src/components/ProductMilestoneDeliverablesAnalysisDetailPage/ProductMilestoneDeliverablesAnalysisDetailPage.tsx

```diff
@@ -17,17 +20,22 @@ import { ContentBox } from 'components/ContentBox/ContentBox';
 import { DateTime } from 'components/DateTime/DateTime';
 import { DeliverablesAnalysisProgressStatusLabelMapper } from 'components/LabelMapper/DeliverablesAnalysisProgressStatusLabelMapper';
 import { DeliverablesAnalysisResultLabelMapper } from 'components/LabelMapper/DeliverablesAnalysisResultLabelMapper';
-import { OldUiContentLinkBox } from 'components/OldUiContentLinkBox/OldUiContentLinkBox';
+import { LogViewer } from 'components/LogViewer/LogViewer';
 import { PageLayout } from 'components/PageLayout/PageLayout';
 import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+import { Toolbar } from 'components/Toolbar/Toolbar';
+import { ToolbarItem } from 'components/Toolbar/ToolbarItem';
 
 import * as operationsApi from 'services/operationsApi';
 import * as productVersionApi from 'services/productVersionApi';
 
+import { timestampHiglighter } from 'utils/preprocessorHelper';
 import { generatePageTitle } from 'utils/titleHelper';
 
```

Comment:
please put it into `common/constants` module and reuse everywhere

Reply:
or you may make it a default value for `delay` param of `useDataBuffer` hook

Reply:
750 was set to default